src/components/Footer/Footer.jsx

```diff
@@ -1,7 +1,138 @@
 import React from 'react';
+import styled from 'styled-components';
+import SOCIAL_LIST from './socialData';
+import INFO_LIST from './infoData';
 
-function Footer(props) {
-  return <div>Footer</div>;
+function Footer() {
+  return (
+    <FooterContainer>
+      <ContentContainer>
+        <Intro>
+          <Title>IOI</Title>
+          <Text>
+            IOI는 모든 사람이 사랑하는 일을 하며 살 수 있도록 세상을 바꾸고자
+            합니다.
+          </Text>
+          <Text>KR 한국어</Text>
+        </Intro>
+        <Content>
+          <Title>About</Title>
+          <Text>홈</Text>
+          <Text>기업교육</Text>
+          <Text>채용</Text>
+          <Text>도움말</Text>
+        </Content>
+        <Content>
+          <Title>크리에이터</Title>
+          <Text>크리에이터 센터</Text>
+          <Text>정규 클래스 지원하기</Text>
+          <Text>도움말</Text>
+        </Content>
+        <Content>
+          <Title>고객센터</Title>
+          <CallBtn>문의하기</CallBtn>
+          <Text>오전 10시 ~ 오후 6시 (주말, 공휴일 제외)</Text>
+        </Content>
+      </ContentContainer>
+      <ContentContainer>
+        <SocialList>
+          {SOCIAL_LIST.map(social => (
+            <WrapText key={social.id}>
+              <TextSmall>{social.title}</TextSmall>
+              <MidDot>&middot;</MidDot>
+            </WrapText>
+          ))}
+        </SocialList>
+        <ListContainer>
+          <InfoList>
+            {INFO_LIST.map(info => (
+              <WrapText key={info.id}>
+                <TextSmall>{info.title}</TextSmall>
+                <MidDot>&middot;</MidDot>
+              </WrapText>
+            ))}
+          </InfoList>
+          <TextSmall>
+            주식회사 IOI | 대표 아이오아이 | 서울특별시 강남구 101 101-101
+            101층(아이오아이 타워) | 사업자등록번호 : 000-00-00000 |
+            통신판매업신고 : 0000-서울강남-0000{' '}
+          </TextSmall>
+        </ListContainer>
+      </ContentContainer>
+    </FooterContainer>
+  );
 }
 
+const FooterContainer = styled.footer`
+  position: fixed;
+  width: 100vw;
+  bottom: 0;
+  border-top: 1px solid #e8e8e8;
+`;
+
+const ContentContainer = styled.div`
+  ${props => props.theme.variables.flex('row', 'center', 'flex-start')}
+  padding: 40px 0 64px 0;
+  width: 100vw;
+`;
+
+const Intro = styled.div`
+  width: 398px;
+`;
+
+const Content = styled.div`
+  width: 240px;
+`;
+
+const Title = styled.p`
+  font-weight: bold;
+`;
+
+const Text = styled.p`
+  margin: 20px 0 0 0;
```

Comment:
```suggestion
  margin-top: 20px;
```
해당 요소의 위에만 margin을 준다고 하면
위와 같은 방법으로 작성하는게 조금 더 직관적이겠네요!